src/load/load.types.ts

```diff
@@ -1,7 +1,22 @@
-export interface ImageMetadata {
+export interface TiffImageMetadata {
   tiff: {
     fields: Map<number, unknown>;
     tags: Record<string, unknown>;
   };
   exif: Record<string, unknown>;
 }
+
+export interface PngImageMetadata {
+  resolution: Resolution;
+}
+
+export type Resolution =
+  | { x: number; y: number; unit: null }
+  | { x: number; y: number }
+  | {
+      x: number;
+      y: number;
+      originalValues: { x: number; y: number; unit: Unit };
+    };
+
+type Unit = 'inch' | 'centimeter';
```

Comment:
Why centimeter? Isn't the other commonly accepted unit the meter?

Reply:
The field does not have to reflect the unit used in the metadata.
One can go back to those if they need to know what the original units were.

For me it seems like both units are useful and we can do a simple conversion already here in image-js. We can generate 2 values, one with the standard metric system in meter, and the other one in inches. WDYT @targos ?

Reply:
> @targos suggest to have 2 normalized fields: One field which resolves the metadata in a normalized way and contains both the original unit and the value. Another computed field (i.e. getter which computes the field when it is accessed) which returns a number in the normalized unit: px/cm
> 
> Can you go along with this?

Yes, but in the issue the normalized unit is pixels per meter. Which one should it be?

Reply:
Made px/cm as normalized unit